test/run.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
```

Comment:
We usually add at least `set -e` in the scripts to prevent uncontrolled command failure.

Reply:
I always use opts from http://redsymbol.net/articles/unofficial-bash-strict-mode/ (although IFS might interfere with some things at times)
```
#!/bin/bash
set -euo pipefail
IFS=$'\n\t'
```

Reply:
done